hazelcast/src/test/java/com/hazelcast/internal/nearcache/AbstractNearCachePreloaderTest.java

```diff
@@ -465,17 +484,21 @@ private static void assertNearCacheContent(NearCacheTestContext<?, ?, ?, ?> cont
             Object nearCacheKey = getNearCacheKey(context, key);
 
             String value = context.serializationService.toObject(getValueFromNearCache(context, nearCacheKey));
-            assertEquals("value-" + i, value);
+            assertEqualsFormat("Expected value %s in Near Cache, but found %s (%s)", "value-" + i, value, context.stats);
 
             assertNearCacheRecord(getRecordFromNearCache(context, nearCacheKey), i, inMemoryFormat);
         }
     }
 
-    // existing preloader test files in src/test/resources/nearcache-* are serialized in BIG_ENDIAN,
-    // therefore it only makes sense to test when serialization service byte order is not overridden to LITTLE_ENDIAN
-    private void assumeBigEndian() {
+    /**
+     * Ensures that the system is running in BIG_ENDIAN byte order.
+     * <p>
+     * The prepared test files in {@code src/test/resources/nearcache-*} are stored in BIG_ENDIAN order, so
+     * we have to skip related tests when the serialization service byte order is overridden to LITTLE_ENDIAN.
+     */
+    private static void assumeBigEndian() {
```

Comment:
Could we move this method to HazelcastTestSupport, where there are other assume methods like for JDKs etc.? I believe it could be useful in the future as well. Also, there might be some overlap with https://github.com/hazelcast/hazelcast/pull/12933/files#diff-44a1455fc6c8f968965f95c3fcc86fa7R212 , just remember that in the back of my head, I'm not saying there is :)

Reply:
I just searched in `HazelcastTestSupport` for such a method, so thanks, good finding! Added another commit to unify the `ByteOrder` checks.